src/data/boundary-landing.json

```diff
@@ -0,0 +1,169 @@
+{
+  "heading": "Boundary Overview",
```

Comment:
For each product, I've added a `src/data/{product}-landing.json` file and updated the contents of the product's page file, to match the existing setups for Vault & Waypoint.